tests/boards/espressif/wifi/src/main.c

```diff
@@ -39,19 +43,19 @@ static void wifi_scan_result(struct net_mgmt_event_callback *cb)
 	uint8_t mac_string_buf[sizeof("xx:xx:xx:xx:xx:xx")];
 	uint8_t ssid_print[WIFI_SSID_MAX_LEN + 1];
 
-	scan_result++;
+	wifi_ctx.scan_result++;
 
-	if (scan_result == 1U) {
+	if (wifi_ctx.scan_result == 1U) {
```

Comment:
Sounds this "scan_result" is only used to print out the 1st scan result. Is that correct? If so, could be boolean instead.

Reply:
It is also used in the table of SSID's (first column), and shows the number of AP's found